action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Install PocketIC server'
+description: 'Install the PocketIC server at a particular version'
+inputs:
+  pocket-ic-server-version:
+    description: >
+      The PocketIC server version to install.
+      If "latest" is provided, the latest version will be installed.
+    default: 'latest'
+
+runs:
+  using: composite
+  steps:
+    - name: Install PocketIC server
+      shell: sh
+      run: |
+        if [ "${{ inputs.pocket-ic-server-version }}" = "latest" ]; then
+          export POCKET_IC_VERSION="6.0.0"
```

Comment:
This needs to be bumped on every release then, right? Is there a way to "look" at the github releases of pocketIC to get the latest version?

Reply:
> This needs to be bumped on every release then, right?

Correct, I've already updated the release doc.

> Is there a way to "look" at the github releases of pocketIC to get the latest version?

maybe this one: https://github.com/marketplace/actions/get-latest-github-release

@marko-k0 do you have a better idea?

Reply:
I don't think you need an action for checking the latest release. You can curl github releases for latest release.

Reply:
> You can curl github releases for latest release

You mean curl and grep in the resulting html?

Reply:
> > You can curl github releases for latest release
> 
> You mean curl and grep in the resulting html?

Yes:
```
curl -s https://api.github.com/repos/dfinity/pocketic/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
```

Reply:
I tried it out, but it was flaky so let's rather hard-code the version: https://github.com/dfinity/pocketic/actions/runs/11232793814/job/31225116735?pr=23